yogstation/code/modules/antagonists/changeling/powers/headcrab.dm

```diff
@@ -0,0 +1,44 @@
+/datum/action/changeling/headcrab
+	name = "Last Resort"
+	desc = "We sacrifice our current body in a moment of need, placing us in control of a vessel that can plant our likeness in a new host. Costs 20 chemicals."
+	helptext = "We will be placed in control of a small, fragile creature. We may attack a corpse like this to plant an egg which will slowly mature into a new form for us. Can be used while unconscious or dead."
+	button_icon_state = "last_resort"
+	chemical_cost = 20
+	dna_cost = 1
+	req_human = 1
+	req_stat = DEAD
```

Comment:
for the last resort change, you don't need to comment out the /tg/ part and you can just do the following in the yogs file:

```
/datum/action/changeling/headcrab
	req_stat = DEAD
```

easy modularization that changes as little as possible